### DISCUSSION TITLE
Recommended hardware

### DISCUSSION THREAD

**Discussion by ddavidebor** [bengtmartensson/IrScrutinizer#427] [Q&A] [answered]
Hi,

Could you commend some hardware for development use to both send and receive?
The most promising seemed IRtoy, but it's always out-of-stock.

What hardware do you feel like has excellent support by irScrutinizer?

Regards,
Davide Bortolami

**Comment by bengtmartensson** [comment, marked as answer]
[this one](http://www.harctoolbox.org/arduino_nano.html).
